src/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php

```diff
@@ -47,6 +47,28 @@ public function register_routes() {
 				'schema' => array( $this, 'get_item_schema' ),
 			)
 		);
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/(?P<name>[\w-]+)',
+			array(
+				'args'   => array(
+					'name' => array(
+						'description' => __( 'Slug name of the theme.' ),
+						'type'        => 'string',
+					),
+				),
```

Comment:
```suggestion
				'args'   => array(
					'stylesheet' => array(
						'description' => __( 'The theme's stylesheet. This uniquely identifies the theme.' ),
						'type'        => 'string',
					),
				),
```

To match the schema.